aws/elastic-beanstalk/elastic-beanstalk.tf

```diff
@@ -0,0 +1,594 @@
+
+variable "solution_stack_name" {
+  default = "64bit Amazon Linux 2018.03 v2.12.14 running Docker 18.06.1-ce"
+  type    = string
+}
+
+variable "env_port_value" {
+  description = "The value to set for the Enviroment variable PORT"
+  default     = "80"
+  type        = string
+}
+
+variable "logs_delete_on_terminate" {
+  description = "Delete cloudwatch logs on termination of beanstalk env"
+  default     = "true"
+  type        = string
+}
+
+variable "app_description" {
+  description = "(Optional) Short description of the application"
+  default     = ""
+  type        = string
+}
+
+variable "asg_min_size" {
+  type        = number
+  default     = 1
+  description = "Minimum number of instances you want in your Auto Scaling group"
+}
+
+variable "asg_max_size" {
+  type        = number
+  default     = 1
+  description = "Maximum number of instances you want in your Auto Scaling group"
+}
+
+variable "asg_availability_zones" {
+  type        = string
+  default     = "any"
+  description = "Availability Zones (AZs) are distinct locations within a region that are engineered to be isolated from failures in other AZs and provide inexpensive, low-latency network connectivity to other AZs in the same region. Choose the number of AZs for your instances"
+}
+
+variable "environment_environment_type" {
+  type        = string
+  default     = "LoadBalanced"
+  description = "Set to SingleInstance to launch one EC2 instance with no load balancer."
+}
+
+variable "environment_load_balancer_type" {
+  type        = string
+  default     = "application"
+  description = "The type of load balancer for your environment"
+}
+
+variable "alb_cert" {
+  description = "Certificate attached to ALB"
+  default     = null
+  type        = string
+}
+
+variable "default_process_port" {
+  type        = number
+  default     = 80
+  description = "Port on which the process listens"
+}
+
+variable "launchconfiguration_ec2_key_name" {
+  type        = string
+  default     = null
+  description = "A key pair enables you to securely log into your EC2 instance"
+}
+
+variable "default_process_protocol" {
+  type        = string
+  default     = "HTTP"
+  description = "Protocol that the process uses. With an application load balancer, you can only set this option to HTTP or HTTPS. With a network load balancer, you can only set this option to TCP."
+}
+
+variable "vpc_id" {
+  default = null
+  type    = string
+}
+
+variable "alb_subnets" {
+  default = null
+  type    = list
+}
+
+variable "ec2_subnets" {
+  default = null
+  type    = list
+}
+
+variable "instances_instance_types" {
+  type        = string
+  default     = "t3.medium"
+  description = "A comma-separated list of instance types you want your environment to use. For example: t2.micro,t3.micro"
+}
+
+variable "aliases" {
+  type        = list(string)
+  description = "List of FQDN's - Used to set the Alternate Domain Names (CNAMEs) setting on Cloudfront"
```

Comment:
What does CloudFront have to do with this? Does EB leverage it?

Reply:
Removed as was not needed